changelog/12663.doc.md

```diff
@@ -1 +1 @@
-Updated commands with square brackets e.g (`pip install rasa[spacy]`) to use quotes (`pip install 'rasa[spacy]'`) for compatibility with zsh in docs.
\ No newline at end of file
+Updated commands with square brackets e.g (`pip install rasa[spacy]`) to use quotes (`pip install 'rasa[spacy]'`) for compatibility with zsh in docs.
```

Comment:
Had to amend this incoming changelog from merging 3.6.x into my branch because of [code quality job failure](https://github.com/RasaHQ/rasa/actions/runs/5532780783/jobs/10095367732#step:15:23).

Reply:
That's so strange, how did the PR pass with that?

Reply:
No idea, added this code: https://github.com/RasaHQ/rasa/pull/12575/commits/efada8b2a3f1d6663410afd4f3b29b12ca4969a1, please re-review 🙏🏻